common/constants/stats.js

```diff
@@ -181,6 +181,60 @@ export const STATS_DATA = {
     suffix: "",
     color: "5",
   },
+  fishing_speed: {
+    name: "Fishing Speed",
+    nameLore: "Fishing Speed",
+    nameShort: "Fishing Speed",
+    nameTiny: "FS",
+    symbol: "☂",
+    suffix: "",
+    color: "b",
+  },
+  breaking_power: {
+    name: "Breaking Power",
+    nameLore: "Breaking Power",
+    nameShort: "Breaking Power",
+    nameTiny: "BP",
+    symbol: "Ⓟ",
+    suffix: "",
+    color: "2",
+  },
+  health_regen: {
+    name: "Health Regen",
+    nameLore: "Health Regen",
+    nameShort: "Health Regen",
+    nameTiny: "HPR",
+    symbol: "❣",
+    suffix: "",
+    color: "c",
+  },
+  vitality: {
+    name: "Vitality",
+    nameLore: "Vitality",
+    nameShort: "Vitality",
+    nameTiny: "Vit",
+    symbol: "♨",
+    suffix: "",
+    color: "5",
+  },
+  mending: {
+    name: "Mending",
+    nameLore: "Mending",
+    nameShort: "Mending",
+    nameTiny: "Mend",
+    symbol: "☄",
+    suffix: "",
+    color: "a",
+  },
+  wisdom: {
+    name: "Wisdom",
+    nameLore: "Wisdom",
+    nameShort: "Wisdom",
+    nameTiny: "W",
+    symbol: "☯",
+    suffix: "",
+    color: "3",
+  },
```

Comment:
Actually can you add a comment to this since ig it's just temporary until #1639 it's merged (with all the real wisdoms)?

like:
`// todo: remove this once #1639 is merged and update wisdom values`